frontend/address-book/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,49 @@
+// import { useState } from "react";
+import {useRecoilState, useSetRecoilState} from "recoil";
+import { searchInputState,currentSearchTerm } from '../states';
+const SearchBar = () => {
+
+    const [searchInput, setSearchInput]= useRecoilState(searchInputState);
+    const [currentSearchValue,setCurrentSearchValue]= useRecoilState(currentSearchTerm );
+    // connect to database for search results?
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        console.log(searchInput);
+        setCurrentSearchValue(searchInput);
+        setSearchInput("");
+    };
+    const button = {
```

Comment:
See? Like here. If you had the css imported from elsewhere, you wouldn't have had to recode it in two spots should the client (aka me) decide they don't like the color scheme.